installer/mac/buildInstaller.sh

```diff
@@ -15,6 +15,12 @@ rm -rf $tempDir
 mkdir $tempDir
 cp -r "./staging/${BRACKETS_APP_NAME}.app/" "$tempDir/$appName"
 
+# copy update.sh to hidden location in the dmg
+if [ -f ./update.sh ]; then
+  echo "Adding update.sh"
```

Comment:
@gavankar, it would be safer to give the file **"executable"** permissions(chmod) programmatically before copying.  Since we would simply be invoking the script later on, we don't want to end up in a situation where the script is present but is not executable due to any number of reasons.

Reply:
Added in commit de725a8